cluster/up.sh

```diff
@@ -19,43 +19,47 @@ set -ex
 source ./cluster/kubevirtci.sh
 CNAO_VERSIOV=0.35.0
 KUBEVIRT_VERSION=v0.29.0
+DEPLOY_KUBEVIRT=${DEPLOY_KUBEVIRT:-true}
 kubevirtci::install
 
-if [[ "$KUBEVIRT_PROVIDER" != external ]]; then
-    $(kubevirtci::path)/cluster-up/up.sh
-fi
+$(kubevirtci::path)/cluster-up/up.sh
```

Comment:
You should remove this line, in case `KUBEVIRT_PROVIDER=external` a cluster will be created

Reply:
yeah it's a duplicate sorry.